spec/features/admin/allocation_chart_spec.rb

```diff
@@ -47,8 +47,10 @@
         end
 
         it '"Alocado até" column links to allocation' do
+          allow(Date).to receive(:today).and_return(Date.new(2023, 4, 1))
+
           within 'tbody' do
-            expect(page).to have_link('01/11/2022', href: "/admin/allocations/#{allocation.id}")
+            expect(page).to have_link('01/03/2023', href: "/admin/allocations/#{allocation.id}")
```

Comment:
After 61 days of the allocation's end date the user will not be shown as `allocated` anymore, so the link that this test is trying to test will not exist. This test will break again in the future because of the date

Reply:
Where is this 61 days rule?

Reply:
`allocation_chart_helper.rb`